apps/www/stories/typography.stories.tsx

```diff
@@ -0,0 +1,161 @@
+import { Meta, StoryObj } from "@storybook/react"
+
+import {
+  BlockQuote,
+  H1,
+  H2,
+  H3,
+  H4,
+  InlineCode,
+  LI,
+  Large,
+  Lead,
+  P,
+  Small,
+  Subtle,
+  TBody,
+  TD,
+  TH,
+  THead,
+  TR,
+  Table,
+  UL,
+} from "@/registry/default/ui/typography"
+
+const meta: Meta = {
+  title: "ui/Typography",
```

Comment:
maybe there should be a folder for theming?  like `theme/typography` which could include ColorPallets as well?

Reply:
Are you kinda thinking of a similar approach to how Figma has its global styles set up? Imo the current implementation using the globals.css in the styles folder works well, making modification of theme colours a fast process. Feel free to correct me if I misunderstood.

Reply:
i was thinking to match more the sidebar structure of the [shadcn/www](https://ui.shadcn.com/docs/theming) with the folders in storybook.  This PR included kind of two features of adding the Typography UI elements as well as supporting storybook so i'm mostly just thinking about where it fits in nicely.  

I could also imagine stories for the colours, paddings, elevation, and borders which are also typical of a design system 🤷  but these don't have to be components

Reply:
Hey friends 👋 

You might be interested in some of our doc blocks if you want to document your design system in Storybook 😁 

- [ColorPalette](https://storybook.js.org/docs/react/api/doc-block-colorpalette)
- [TypeSet](https://storybook.js.org/docs/react/api/doc-block-typeset)

We don't currently have anything for spacing, elevation, or borders but it wouldn't be too difficult to put together a couple of components to do that.

I wrote [something](https://github.com/Integrayshaun/design-tokens-sandbox) a little while ago to read design token files and generate design system doc pages in Storybook that you could use as inspiration.

![image](https://github.com/shadcn/ui/assets/18172605/11c33b7e-c7ef-4aa2-b744-1b28db6f570c)
![image](https://github.com/shadcn/ui/assets/18172605/db1390a2-2942-450a-9eac-fe95992591be)
![image](https://github.com/shadcn/ui/assets/18172605/b9d252bd-b352-4073-913e-db334ea7baf1)

Reply:
> I wrote [something](https://github.com/Integrayshaun/design-tokens-sandbox) a little while ago to read design token files and generate design system doc pages in Storybook that you could use as inspiration.

This is amazing!  I'm actually just in the process of building out a token-to-tailwind plugin using style-dictionary so looking how you've done it in this repo is quite amazing and wonderful reference!

I might be able to adapt the tailwind.config.js to work directly in MDX, for this PR, but for sure will try make something ColorPallete and Typeset block 👍